pkgs/top-level/python-packages.nix

```diff
@@ -20204,6 +20204,26 @@ EOF
 
   pypandoc = callPackage ../development/python-modules/pypandoc { };
 
+  mdv = buildPythonPackage rec {
+    pname = "mdv";
+    name = "${pname}-${version}";
+    version = "1.6.3";
+
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/${strings.substring 0 1 pname}/${pname}/${name}.tar.gz";
+      sha256 = "1x4d1gyvmdqh2zpdc415ak10d4yfli04pw4mxwfv2jnjk94d1y0k";
+    };
+
+    propagatedBuildInputs = with self; [ markdown pygments pyyaml docopt ];
+
+    meta = {
+      homepage = https://github.com/axiros/terminal_markdown_viewer;
+      description = "Terminal Markdown Viewer";
+      license = licenses.bsd;
+      maintainers = with maintainers; [ synthetica ];
```

Comment:
You need to add yourself to https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix.